packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/process/schema.ts

```diff
@@ -40,10 +40,12 @@ export const QueryByParameterSchema = z.object({
         semanticsText: z.literal("LivingSubject.id"),
       })
       .optional(),
-    livingSubjectName: z.object({
-      value: schemaOrArray(nameSchema),
-      semanticsText: z.literal("LivingSubject.name"),
-    }),
+    livingSubjectName: schemaOrArray(
+      z.object({
+        value: nameSchema,
+        semanticsText: z.literal("LivingSubject.name").optional(),
+      })
+    ),
```

Comment:
`{
    "code": "invalid_type",
    "expected": "object",
    "received": "array",
    "path": [
        "Envelope",
        "Body",
        "PRPA_IN201305UV02",
        "controlActProcess",
        "queryByParameter",
        "parameterList",
        "livingSubjectName"
    ],
    "message": "Expected object, received array"
}`

Fix for this schema error. LivingSubjectName can be an array while value is only allowed to be a single object
https://profiles.ihe.net/ITI/TF/Volume2/ITI-55.html